shared/video-sdk/develop/stream-raw-audio-and-video/project-implementation/windows.mdx

```diff
@@ -301,15 +318,19 @@ This sections describes the steps required to use the relevant libraries and dec
         }
         ```
 
-    1. To register the observers at startup, in `AgoraImplementationDlg.cpp`, add the following at the of `setupVideoSDKEngine()`:
+    1. To register the observers at startup, in `AgoraImplementationDlg.cpp`, add the following at the start of
+  `setupVideoSDKEngine()`:
 
         ```cpp
         // Enable and register observers for audio and video frames
         EnableAudioVideoCapture(TRUE);
         ```
 3. **Unregister the video and audio frame observers when you close the <Vpl k="CLIENT"/>**
     
-    When you quit the <Vpl k="CLIENT"/>, unregister the frame observers by disabling the `EnableAudioVideoCapture()` method with a `false` argument. To implement this workflow, in `AgoraImplementationDlg.cpp`, add the following to `OnClose` before `	agoraEngine->release();`:
+    When you quit the <Vpl k="CLIENT"/>, to unregister the frame observers, set
```

Comment:
Again It should only be at the 'LEAVE' button listener , not in `OnClose` . Suppose in a system if user can join. I am not convinced with the idea of disabling the frame observer in `OnClose()`